### ISSUE TITLE
Consider adding Trash

### ISSUE THREAD

**Issue by thejimbirch** [phenaproxima/starshot-prototype#83]
WordPress has a trash system that allows content editors to restore entities after they have been deleted.

I just learned of the Drupal Trash module and after trying it out, I am sold.  I want to install this on every site I have ever built.

https://www.drupal.org/project/trash

Caveat that there is an open issue for Content Moderation support.

https://www.drupal.org/project/trash/issues/3320566

**Comment by pameeela** [comment]
This does sound awesome, but I think the content moderation issue means we can't add it yet, since moderation is enabled by default. But maybe we can get some more eyes on that issue?

**Comment by phenaproxima** [comment]
Yeah, if it doesn't work with Content Moderation I think that we couldn't merge this just yet. :(

Although...if that issue has a patch, we could add Trash to Starshot with that patch applied. But it would be preferable to fix the issue upstream and tag a new release.

**Comment by mlncn** [comment]
[Workflow buttons](https://www.drupal.org/project/workflow_buttons) has a "Trash workflow" submodule that is intended to provide the same functionality.  We would be up for some work to get this into Starshot— provided that the base module's main enhancement, turning the unwieldy moderation state dropdown into beautiful buttons, is also wanted!
